core/src/v2/actix.rs

```diff
@@ -155,6 +155,8 @@ mod manual_impl {
     impl_simple!(usize);
     #[cfg(feature = "datetime")]
     impl_simple!(chrono::NaiveDateTime);
+    #[cfg(feature = "datetime")]
+    impl_simple!(chrono::DateTime<chrono::Utc>);
```

Comment:
Can we implement this for generic `DateTime<T>` rather than just for `DateTime<Utc>`?